### ISSUE TITLE
O> Issues list maintenance by guides

### ISSUE THREAD

**Issue by dckc** [rchain/bounties#401]
> Bounty task guides for a number of areas are here to align contributions with goals of the RChain cooperative -- [Bounty Task Guides](https://github.com/rchain/bounties/wiki/Bounty-Task-Guides)

_unguided issue query has been moved to that wiki page_.

Measure of Completion:  issues are regularly reviewed and pruned by a collection of guides, preferably with some overlap / redundancy.

**Comment by zsluedem** [comment]
What about the finished issue without paying bounties? 
I suggest we create a new label for those issues. It would help manage those bounty issue at the end of the month.

**Comment by dckc** [comment]
@zsluedem elaborate, please? I don't understand. What is the situation that you are concerned about? What steps lead up to it?

Each person is responsible for collecting their own bounties by voting on budgets and rewards. 

I encourage rewards only for closed issues (#318) and I encourage others to do likewise, but it's not something I want added to the `CONTRIBUTING.md` bounty contract as a hard-and-fast rule at this time.

**Comment by zsluedem** [comment]
@dckc OK, I got your meaning. We see this in different perspective. I think the issue should be closed when it is done paying the rewards.But there are always some issues without any bounties. So I think  what you suggest reward for closed issues is cool.

Because I think it's kind of hard for one to organize all of the bounties at the end of the month, I think the new label of closed issue would help.However, the bounty app will also solve this problem. So I was over-worried.

**Comment by lapin7** [comment]
When there's a reward on a closed issue for February, the reward will be also on the invoice of February. So it would get paid.

**Comment by dckc** [comment]
I see somebody put some of the guides in the label descriptions. Thanks!

**Comment by lapin7** [comment]
Yep. Only a url is missing to contributing.md

**Comment by dckc** [comment]
@ICA3DaR5 you're already doing most of what I have in mind. I suppose I didn't explain what I meant by pruning:

  - Use **invalid** or **wontfix** labels to close issues that are no longer worth tracking.
  - label issues as **needs-SMART-objective** as a warning if, after a week or two to allow for brainstorming, there isn't a plausible plan in the issue or at least a clear next step.

**Comment by ICA3DaR5** [comment]
@dckc Voted on this, as well.

**Comment by dckc** [comment]
I'm still looking for confirmation from guides for Operations, Marketing, and Governance.

**Comment by pmoorman** [comment]
Happy to take care of the marketing section, @dckc @patrick727

**Comment by dckc** [comment]
Excellent, @pmoorman . I see a lot of work to do in that area. Last time I checked there were 61 open Marketing issues; I doubt they're all still worth tracking.

**Comment by pmoorman** [comment]
Agreed!  I plan to go through them tomorrow or the day after. There's definitely a bit of cleanup needed

**Comment by dckc** [comment]
@ICA3DaR5 training people to use the new issue template for translation looks like non-trivial work; please give the author of #453 a nudge.

**Comment by pmoorman** [comment]
Just went through the **marketing issues,** and maybe it's good to report some basic findings:

- I was surprised how many issues from more than 3-4 months were still open

- A great many issues that are longstanding open (and mostly inactive) have to do with branding and clearly organizing/consolidating templates and other marketing "building blocks" for marketing efforts.  
_=>  I hope that the branding WG will address most of these, as apparently the bounty system is not the right way to resolve them._

- It seemed to me -- but this is gut feeling -- that in the last month or so, much _fewer_ marketing issues have been opened than before

- It seems like many marketers are uncomfortable with the SMART definition of a scope. I think we should all help each other here -- where possible -- to suggest improvements.

**Comment by dckc** [comment]
> I hope that the branding WG will address most of these, as apparently the bounty system is not the right way to resolve them.

I hope those are not exclusive. I hope the branding WG can make use of the bounty system.

> I was surprised how many issues from more than 3-4 months were still open

I'm a little bit disappointed, but I'm not at all surprised. Systems naturally tend toward entropy; it takes work to bring order. 3-4 months ago the whole system was new and everybody was just figuring it out.

Keep up the good work!

**Comment by dckc** [comment]
I added a few labels to the query:
 - [is:issue is:open -label:Translation -label:Marketing -label:Development -label:Operations -label:Governance -label:member-site -label:nom-committee -label:greeter](https://github.com/rchain/bounties/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3ATranslation+-label%3AMarketing+-label%3ADevelopment+-label%3AOperations+-label%3AGovernance+-label%3Amember-site+-label%3Anom-committee+-label%3Agreeter)

There are three "stray" issues from this perspective. I don't see any particular way to move them forward, but I think 3 is a small enough number to leave them as they are.

**Comment by dckc** [comment]
@pmoorman are you involved in coordinating RChain speakers at blockchain events? Please take a look at #583 and #593.

**Comment by dckc** [comment]
I moved #583 (along with #593) from **Marketing** to **Events**.

Anybody want to guide **Events** issues?

**Comment by Ojimadu** [comment]
Sure, I can guide issues surrounding events and occasions.

**Comment by dckc** [comment]
Thanks for stepping up, @Ojimadu . Are you in contact with the people who would be speaking at the events? Or can you think of some other way to coordinate?

There's a **#pr-opportunities** channel in discord. I would expect considerable overlap between discussion of which PR opportunities to pursue and which speaking opportunities to pursue. Do you know how #pr-opportunities get evaluated?

**Comment by Ojimadu** [comment]
Speakers would depend on what the coop is trying to achieve from the event. I had once evaluated getting speaking positions on behalf of the coop but that seems delicate as the speakers would need to be contacted on their schedule and time. Also, there is a question as to what capacity a given speaker would speak -as a coop member or as an official representative of the co-op -.

For the coop and its partners, getting a list of events that they would be interested in attending and pitching it to them IMO would be better as I believe these entities would be able to get organize speakers if otherwise, they might seek help from the coop members.
For other individuals that might come up to speak at events, depending on how familiar they are with RChain would probably have their text reviewed to avoid misrepresentation. 
@makys raised an [important point](https://github.com/rchain/bounties/issues/241#issuecomment-361159607) on issue #241  which is worth considering.

**Comment by TrenchFloat** [comment]
I frequently check the Operations and Education labels. I can be the guide for those labels.

**Comment by dckc** [comment]
I just made a [Bounty Task Guides](https://github.com/rchain/bounties/wiki/Bounty-Task-Guides) wiki page, which consolidates info from this issue and #616 on the TOS.

I hope to see regular confirmation from guides that they are serving in this role.

**Comment by dckc** [comment]
Your Mar 8 "Just went through the marketing issues ..." comment is perhaps archtypical of regular confirmation.

But I can imagine other forms...

Perhaps notes from a Gov WG meeting would show Alex (pretend name) saying "I took my turn reviewing Governance issues this week. I closed a couple stinky ones. The new ones were straightforward to give guidance except for two that I'd like your input on..." and the group goes on to discuss them, resulting in feedback comments in the issues.

Or maybe there are only 7 **greeter** issues and it's self-evident that they're in good order. It's slightly more convenient if somebody shows and tells us that things are in order, but if it's plain at a glance, then all is well.
